processor/resourcedetectionprocessor/internal/aws/elasticbeanstalk/config.go

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package elasticbeanstalk // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor/internal/aws/elasticbeanstalk"
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor/internal/aws/elasticbeanstalk/internal/metadata"
+)
+
+// Config defines user-specified configurations unique to the EC2 detector
+type Config struct {
+	// Tags is a list of regex's to match ec2 instance tag keys that users want
+	// to add as resource attributes to processed data
```

Comment:
copy-paste artifact :)

Reply:
Yeah, I did plenty of copy-pasting here 😅 I deleted comments from every detector that has only this in the config as I think it is self-explanatory. Let me know if you agree.